tools/codegen/api/lazy.py

```diff
@@ -242,3 +245,71 @@ def keyword_values(self) -> List[LazyArgument]:
     @property
     def keyword_scalars(self) -> List[LazyArgument]:
         return self.filtered_args(positional=False, keyword=True, values=False, scalars=True)
+
+class NonNativeLazyIrSchema:
+    def __init__(self, name: str, op: Dict[str, Any]):
+        self.num_inputs = int(op.get("num_inputs", 1))
+        self.num_outputs = int(op.get("num_outputs", 1))
+        self.has_shape = op.get("has_shape", True)
+        self.vars = op.get("vars", {})
+
+        self.name = name
+        self.inputs = [f"input{i}" for i in range(self.num_inputs)]
+        self.returns = [f"output{i}" for i in range(self.num_outputs)]
+
+        self.aten_name = op["sym"]
+        if not self.aten_name.startswith("at::"):
+            self.ltc_name = self.aten_name
+
+        args: List[Namespace] = []
```

Comment:
is Namespace being used in its intended way here? I wonder why this isn't `List[LazyArgument]` like other LazySchemas

Reply:
> is Namespace being used in its intended way here?

Not exactly. The reason I didn't use `LazyArgument` here is because I thought it would be more difficult to construct one than to just fill in the fields that I needed using an arbitrary object.

I can try to make it a `LazyArgument` if you'd prefer